spec/takeaway_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'takeaway'
+
+describe Takeaway do
+  # describe '#show_menu' do
+  #   # subject(:menu) { Menu.}
```

Comment:
The reason using `subject` probably didn't work is that it's an RSpec that refers to the class mentioned on line 3 after the `describe` keyword (so that would be `Takeaway`) in this case. 

What you want here is to create a menu [double](https://github.com/makersacademy/course/blob/main/pills/doubles.md#doubles-mocks-stubs-spies-and-fakes) to represent an instance of the Menu class. The simplest way of creating a double is like this:
```
my_double = double()
```
You can also give your double a name to make it easier to identify in any test output

```
my_double = double(:menu_double)
```

You'll also have tell the double that it should respond to the `show_menu` method and return something. The examples on this page might help: https://relishapp.com/rspec/rspec-mocks/v/3-10/docs/basics/partial-test-doubles#only-the-specified-methods-are-redefined/

Then you'd have to pass your double into the `initialize` method of your Takeaway.